<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modNaprawa</class>
 <widget class="QDialog" name="modNaprawa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>351</width>
     <height>229</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="numerFakturyLabel">
      <property name="text">
       <string>Numer Faktury</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="numerFakturyLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="dataWykonaniaLabel">
      <property name="text">
       <string>Data Wykonania</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="dataWykonaniaDateEdit">
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="warsztatLabel">
      <property name="text">
       <string>Warsztat</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="numerRejestracyjnySamochoduLabel">
      <property name="text">
       <string>Nr rej. samochodu</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPlainTextEdit" name="opisTextEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="kosztLabel">
      <property name="text">
       <string>Koszt</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="kosztDoubleSpinBox">
      <property name="prefix">
       <string notr="true"/>
      </property>
      <property name="suffix">
       <string notr="true"/>
      </property>
      <property name="maximum">
       <double>999999.989999999990687</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Opis:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="numerRejestracyjnyBox"/>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="warsztatBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>389</y>
     <width>194</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="Ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="anuluj">
      <property name="text">
       <string>Anuluj</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>354</width>
     <height>146</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Wybierz numer rejestracyjny samochodu:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Wybierz numer faktury naprawy, którą chcesz zmodyfikować:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Wybierz warsztat w którym dokonano naprawy</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="nrFakCBwybor"/>
    </item>
    <item row="4" column="0">
     <widget class="QComboBox" name="nrRejCBwybor"/>
    </item>
    <item row="6" column="0">
     <widget class="QComboBox" name="warsztatCBwybor"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
